fhirspec.py

```diff
@@ -74,7 +74,10 @@ def read_valuesets(self):
                 self.valuesets[resource['url']] = FHIRValueSet(self, resource)
             elif 'CodeSystem' == resource['resourceType']:
                 assert 'url' in resource
-                self.codesystems[resource['url']] = FHIRCodeSystem(self, resource)
+                ## TODO Need to remove this to think of a better way to handle system with no codes.
+                ## http://hl7.org/fhir/2018Jan/valueset-operation-outcome.html <- bad build of the FHIR spec for R4 snapshot
+                if len(self.codesystems) > 0:
+                    self.codesystems[resource['url']] = FHIRCodeSystem(self, resource)
```

Comment:
Can you add an `else: logger.warn("ValueSet with 0 codes: {}".format(resource['url']))`?